finalfrontier-utils/src/util.rs

```diff
@@ -65,15 +65,18 @@ impl SkipGramApp {
                 Arg::with_name(CONTEXT)
                     .long("context")
                     .value_name("CONTEXT_SIZE")
-                    .help("Context size (default: 5)")
-                    .takes_value(true),
+                    .help("Context size")
+                    .takes_value(true)
+                    .default_value("10"),
             )
             .arg(
                 Arg::with_name(MODEL)
                     .long(MODEL)
                     .value_name("MODEL")
-                    .help("Model: skipgram or structgram")
-                    .takes_value(true),
+                    .help("Model")
+                    .takes_value(true)
+                    .possible_values(&["skipgram", "structgram"])
```

Comment:
So later maybe there will be the third option dirgram

Reply:
Yes. But I can rebase this PR once your PR is merged and add the change.

Reply:
Oh and maybe we need to change the readme page too?

Reply:
And the man page, see #50